### ISSUE TITLE
Add support for module specifier "aliases" (tsconfig.paths mappings) when moving files/folders.

### ISSUE THREAD

**Issue by tryggvigy** [dsherret/ts-simple-ast#1494]
**tl;dr** Add support for module specifier "aliases" (tsconfig.paths mappings) when moving files/folders.

---

Let's say your tsconfig contains a paths mapping like this:
```
    "paths": {
      "Root/*": ["./*"]
    },
```

I noticed that 

```ts
project.createDirectory('someDir');
const srcFile = project.getSourceFileOrThrow('someFile.ts');
srcFile.moveToDirectory('someDir');
```

will update all relative import/export specifiers referencing 'someFile.ts' but _not_ if the specifier used an alias such as 

```ts
import * from 'Root/someFile';
```

I expected `ts-morph` to update the specifier to `import * from 'Root/someDir/someFile';` but it doesn't.
I think it would be nice feature to support this. VSCode can handle paths mapped import/exports when moving files.

**Describe the solution you'd like**

I think the problem is that [`updateStringLiteralReferences(nodeReferences: ReadonlyArray<[StringLiteral, SourceFile]>) {`](https://github.com/dsherret/ts-morph/blob/930779a0d3699be35ab17bbe9dd26ec00350fe53/packages/ts-morph/src/compiler/ast/module/SourceFile.ts#L1025C1-L1028C4) only updates the module specifier if it's a relative path like `'../someFile'` not if it's an module specifier alias like `'Root/someFile'`.
